awscli/handlers.py

```diff
@@ -102,6 +103,7 @@ def awscli_initialize(event_handlers):
                             ec2_add_priv_launch_key)
     register_parse_global_args(event_handlers)
     register_pagination(event_handlers)
+    event_handlers.register('operation-args-parsed.ec2.*', set_max_results_default)
```

Comment:
Should we be doing this for every ec2 operation? Does every ec2 operation have the "paginate and give me a pagination token only if you gave me a page size to start"?